ckanext/opendata_theme/opengov_custom_css/templates/base.html

```diff
@@ -1,9 +1,9 @@
 {% ckan_extends %}
 
 {% block styles %}
-  {{ super() }}
+    {{ super() }}
     <style>
-        {% resource 'opengov_custom_css_resource/css/theme.css' %}
-        {{ g['ckanext.opendata_theme.custom_raw_css'] | safe }}
+        {% set _type = 'asset' if h.ckan_version() > '2.9' else 'resource'  %}
```

Comment:
The check may fail if version `2.10` is evaluated as `2.1`.

Reply:
fixed